src/components/CvViewer/Templates/Basic/Basic.tsx

```diff
@@ -12,33 +12,39 @@ type CvViewerProps = {
 };
 
 export const Basic = ({ cv, isReverse = false }: CvViewerProps) => {
-  const { personalInfo, workExperience, education, skills, hobbies, additionalBlocks } = cv;
+  const { personalInfo, workExperience, education, skills, hobbies } = cv;
 
   return (
     <div className={isReverse ? css.wrapperReverse : css.wrapper}>
       <div className={css.leftBlock}>
         <TextWithHeading heading="Description" text={personalInfo.description} />
         <TextWithHeading heading="Work experience" />
         {(workExperience || []).map((el) => (
-          <CVExperienceBlock key={el.companyName} {...el} />
+          <CVExperienceBlock
+            key={el.companyName}
+            title={el.companyName}
+            subtitle={el.position}
+            startDate={el.dateTime?.[0]}
+            endDate={el.dateTime?.[1]}
+            description={el.description}
```

Comment:
Oh, now I see that here you pass description as an AI answer, so in such case it could be better to rename in state this name to something like we had before with sufix AI, so that you will understand right from the name that this is the place where we keep the answer from an AI. So for me it seem more obvious to rename remark to description and description to answerFromAI or smth

Reply:
Yeap, make sense